app/cmd/plugin_search.go

```diff
@@ -72,19 +72,119 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 	return result
 }
 
+func matchPluginsData(plugins []client.AvailablePlugin, pluginJclient *client.PluginManager) (result []client.CenterPlugin) {
+	if len(plugins) == 0 {
+		return
+	}
+	result = make([]client.CenterPlugin, 0)
+	jclient := &client.UpdateCenterManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(jclient.JenkinsCore))
+	site, err := jclient.GetSite()
+	noSite := false
+	if err != nil && site == nil {
+		noSite = true
+	}
+	installedPlugins, err := pluginJclient.GetPlugins()
+	noInstalledPlugin := false
+	if err != nil && installedPlugins == nil {
+		noInstalledPlugin = true
+	}
+	for _, plugin := range plugins {
+		result = buildData(noSite, site, plugin, result, noInstalledPlugin, installedPlugins)
+	}
+	return
+}
+
+func buildData(noSite bool, site *client.CenterSite, plugin client.AvailablePlugin, result []client.CenterPlugin, noInstalledPlugin bool, installedPlugins *client.InstalledPluginList) (resultData []client.CenterPlugin) {
+	isTrue := false
```

Comment:
`isTrue`? What is true?

Reply:
It might be better to change to `isMatched`. Isn't it?